src/tribler/core/utilities/path_util.py

```diff
@@ -36,8 +38,23 @@ def normalize_to(self, base: str = None) -> Path:
 
         return self
 
-    def size(self) -> int:
-        return self.stat().st_size
+    def size(self, only_files: bool = False) -> int:
+        """ Return the size of this file or directory (recursively).
+
+        Args:
+            only_files: If True, only return the size of files in this directory, not the size of files+directories.
```

Comment:
The `only_files` option looks a bit confusing. At the first read, I interpreted it as "calculate only the size of files in the top-level-directory and not in subdirectories" and thought it was better to name it `include_subdirectories=True` instead of `only_files=False`.

But then I realized that it means the size of directory entries that are often missed from the total size.

I like the idea of taking into account the size of directory entries, but I suspect that for other developers in the future, it also may not be totally obvious what this option is doing. Can we rename it to something like `include_directory_entry_sizes=True` or `include_dir_sizes=True`?

Reply:
Sure

Reply:
Done